R/pkg/NAMESPACE

```diff
@@ -224,7 +224,8 @@ exportMethods("agg")
 export("sparkRSQL.init",
        "sparkRHive.init")
 
-export("cacheTable",
+export("as.DataFrame",
```

Comment:
This is not a method of DataFrame. This is a method to create a DataFrame. Export it adjacent to "createDataFrame"